pkg/virt-handler/non-root.go

```diff
@@ -167,6 +167,32 @@ func (d *VirtualMachineController) prepareTap(vmi *v1.VirtualMachineInstance, re
 
 }
 
+// FindInterfaceIndexPath return the ifindex path of the given interface name (e.g.: enp0f1p2 -> /sys/class/net/enp0f1p2/ifindex).
+// If path not found it will try to find the path using ordinal pod interface name based on its position in the given
+// networks slice (e.g.: net1 -> /sys/class/net/net1/ifindex).
+func FindInterfaceIndexPath(res isolation.IsolationResult, podIfaceName string, networkName string, networks []v1.Network) (*safepath.Path, error) {
```

Comment:
In the context of this function, it is not a "pod" interface, it is just an interface.